### ISSUE TITLE
Unable to calculate thermal conductivity for parahydrogen

### ISSUE THREAD

**Issue by Sum4691** [CoolProp/CoolProp#1851]
Unable to calculate thermal conductivity for parahydrogen when the temperature is lower than 49.41 K, while the Refprop can.

**Comment by Sum4691** [comment]
Here is the Pic, the unit for the first row is MPa.
![Pic](https://user-images.githubusercontent.com/54394412/63501002-762de500-c4fd-11e9-83a9-cdcc2970bf57.PNG)

**Comment by ibell** [comment]
What's the error?  There should be a message in the cell.

Please fill out the issue template in full.

**Comment by Sum4691** [comment]
I’m sorry.

### Description

I’m unable to calculate thermal conductivity for parahydrogen when the temperature is lower than 49.41 K
### Steps to Reproduce

1.	Using the Excel formula “=PropsSI("L","P",F$1*1000000,"T",$E4,"parahydrogen")” to calculate the thermal conductivity for parahydrogen.
2.	The pressure is range from 0.1 to 1 MPa and the temperature is range from 20 to 51 degC.

**Expected behavior:** [What you expect to happen]
Showing the correct values.
**Actual behavior:** [What actually happens]
 
#VALUE! error

**CoolProp Version:**V6.3.0 32bit  
**Operating System and Version:** Windows 7  
**Access Method:**Excel

![图片](https://user-images.githubusercontent.com/54394412/63564802-1e44bc00-c599-11e9-9709-103f335319ac.png)

**Comment by ibell** [comment]
Perfect, there is your answer: there is no model in CoolProp for parahydrogen's thermal conductivity.

**Comment by Sum4691** [comment]
OK.
But isn't that a problem? Because the RefProp is capable of such calculation.
![2](https://user-images.githubusercontent.com/54394412/63565865-5fd76600-c59d-11e9-85e9-f4b3f2aed38e.PNG)

**Comment by ibell** [comment]
Yes, but for CoolProp to get this functionality, somebody has to do this.

**Comment by tnorby** [comment]
It looks like the viscosity model got lost since http://www.coolprop.org/v4/_static/doxygen/html/_hydrogen_8cpp_source.html#l00430

Note: "viscosity" key/value missing in .json
[HvsParaH.pdf](https://github.com/CoolProp/CoolProp/files/5446368/HvsParaH.pdf)

**Comment by imkelisa** [comment]
I have the same problem, I would like to use the Parahydrogen model, in the excel interface I get the error message that 

**Viscosity** model is   not available for this fluid :   PropsSI("L","T",21.5,"P",101325,"ParaHydrogen") although I am looking for the **heat conductivity**. In the fluid list, a reference paper is listed for calculation of heat property of parahydrogen. Would it at least be possible to deactivate the reference in the list?
